streams/src/main/java/org/apache/kafka/streams/processor/internals/GlobalStreamThread.java

```diff
@@ -138,18 +143,18 @@ public State state() {
      * Sets the state
      * @param newState New state
      */
-    private boolean setState(final State newState) {
+    private void setState(final State newState) {
```

Comment:
why this change?

Reply:
It's a private method with an unused return value. Making it void helps the reader to understand the code without having to trace through usages.

Reply:
@vvcephei @bbejeck @mjsax I'd suggest in the future try to only piggy-back different changes into the same PR if we think they are either correlated or if they are really trivial. Having a single PR mingled with multiple changes has several drawbacks:

1. It makes git history a bit harder to trace: think, "git blame" would be tricker to reason.
2. It tends to generate bigger PRs than necessary, making reviewer less willing to start working on them :P
3. If multiple rounds of reviews are needed, even requiring major code refactoring, it will surprisingly introduce regressions during those iterations as by-products of the multiple changes.